eng/common/pipelines/templates/steps/prepare-pipelines.yml

```diff
@@ -17,171 +17,176 @@ parameters:
-        --patvar PATVAR
-        --set-managed-variables
-        --debug
-        ${{parameters.UPConventionOptions}}
-      displayName: Create UP Pipelines for Public Repository
-      env:
-        PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
-    - script: >
-        $(Pipeline.Workspace)/pipeline-generator/pipeline-generator
-        --organization https://dev.azure.com/azure-sdk
-        --project internal
-        --prefix ${{parameters.Prefix}}
-        --devopspath "\${{parameters.Prefix}}"
-        --path $(System.DefaultWorkingDirectory)/sdk
-        --endpoint Azure
-        --repository ${{parameters.Repository}}
-        --convention tests
-        --agentpool Hosted
-        --branch refs/heads/$(DefaultBranch)
-        --patvar PATVAR
-        --set-managed-variables
-        --debug
-        ${{parameters.TestsConventionOptions}}
-      displayName: Create Live Test Pipelines for Public Repository
-      condition: and(succeeded(), ne('${{parameters.TestsConventionOptions}}',''))
-      env:
-        PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
-    - script: >
-        $(Pipeline.Workspace)/pipeline-generator/pipeline-generator
-        --organization https://dev.azure.com/azure-sdk
-        --project internal
-        --prefix ${{parameters.Prefix}}
-        --devopspath "\${{parameters.Prefix}}"
-        --path $(System.DefaultWorkingDirectory)/sdk
-        --endpoint Azure
-        --repository ${{parameters.Repository}}
-        --convention testsweekly
-        --agentpool Hosted
-        --branch refs/heads/$(DefaultBranch)
-        --patvar PATVAR
-        --set-managed-variables
-        --debug
-        ${{parameters.TestsConventionOptions}}
-      displayName: Create Weekly (Multi-Cloud) Live Test Pipelines for Public Repository
-      condition: and(succeeded(), ne('${{parameters.TestsConventionOptions}}',''))
-      env:
-        PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
-    - script: >
-        $(Pipeline.Workspace)/pipeline-generator/pipeline-generator
-        --organization https://dev.azure.com/azure-sdk
-        --project internal
-        --prefix ${{parameters.Prefix}}
-        --devopspath "\${{parameters.Prefix}}"
-        --path $(System.DefaultWorkingDirectory)/sdk
-        --endpoint Azure
-        --repository ${{parameters.Repository}}
-        --convention upweekly
-        --agentpool Hosted
-        --branch refs/heads/$(DefaultBranch)
-        --patvar PATVAR
-        --set-managed-variables
-        --debug
-        ${{parameters.UPConventionOptions}}
-      displayName: Create Weekly (Multi-Cloud) Unified Test Pipelines for Public Repository
-      condition: and(succeeded(), eq(${{parameters.GenerateUnifiedWeekly}},true))
-      env:
-        PATVAR: $(azuresdk-azure-sdk-devops-pipeline-generation-pat)
+jobs:
+- job: PreparePipelines
+  pool:
+    name: azsdk-pool-mms-ubuntu-2004-general
+    vmImage: MMSUbuntu20.04
```

Comment:
We will want this to be moved to 22.04 once we fix the tool to run on .NET 6.0

Reply:
Mentioned this in #4930